ui/packages/catalog-ui-search/src/main/webapp/component/map-context-menu/map-context-menu.less

```diff
@@ -1,59 +1,4 @@
-@{customElementNamespace}map-context-menu {
-  .metacard-interaction,
-  .interaction-icon,
-  .interaction-text {
-    line-height: @minimumButtonSize;
-    height: auto;
-  }
-  .metacard-interaction {
-    white-space: nowrap;
-    padding: 0px 10px;
-    cursor: pointer;
-    overflow: hidden;
-  }
-  .interaction-icon {
-    text-align: center;
-    width: @minimumButtonSize;
-  }
-  .interaction-text {
-    max-width: ~'calc(100% - @{minimumButtonSize})';
-    > div {
-      overflow: hidden;
-      text-overflow: ellipsis;
-    }
-  }
-  .interaction-icon,
-  .interaction-text {
-    display: inline-block;
-    vertical-align: middle;
-  }
-  .interaction-view-details,
-  .interaction-view-details-selection,
-  .interaction-view-histogram-selection {
-    display: none;
-  }
-  .fa-chevron-withmargin {
-    margin-left: 10px;
-  }
-}
-
-@{customElementNamespace}map-context-menu.is-off-map {
-  .interaction-copy-coordinates,
-  .interaction-copy-dms,
-  .interaction-copy-wkt {
-    display: none;
-  }
-}
-
-@{customElementNamespace}map-context-menu.has-target {
-  .interaction-view-details {
-    display: block;
-  }
-}
-
-@{customElementNamespace}map-context-menu.has-selection {
-  .interaction-view-details-selection,
-  .interaction-view-histogram-selection {
-    display: block;
-  }
+body div intrigue-map-context-menu .input-menu-item {
```

Comment:
This is the last css rule in this file, is there a way to delete it completely?

Reply:
I really wanted too, unfortunately this css affects other components. Until those are replaced with React components, then css is the only way to control them from here.

Reply:
You should still be able to keep `@{customElementNamespace}`.